### PR TITLE
Added deletion of events

### PR DESCRIPTION
**Pull request by sarahcxh**
Added deletion of events. 
Added in code for auto-deletion of events as the date passes but buggy so I commented it out. Will fix it.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteEventCommand.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+public class DeleteEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "(ev)delete";
+    public static final String COMMAND_FUNCTION = "Deletes the event identified by the "
+            + "index number used in the displayed event schedule.";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": " + COMMAND_FUNCTION + "\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_EVENT_SUCCESS = "Deleted Event: %1$s\nYou now have one less "
```
Comment (ElysiaTanZY) [inline src/main/java/seedu/address/logic/commands/DeleteEventCommand.java:25]: Might want to change the message here cause cause events are not always work related so saying 'Keep up the good work' might be a bit off O: HAHA

File: src/main/java/seedu/address/logic/commands/ListEventCommand.java [hunk 2]
```diff
@@ -31,9 +32,29 @@ public ListEventCommand(Comparator<Event> comparator) {
...
+
+        /*
+        int length = model.getEventsList().size();
+        for (int i = 0; i < length; i++) {
+            Event event = model.getEventsList().get(i);
+            if (isOverdue(event)) {
+                model.deleteEvent(event);
+            }
+        }
+        */
```
Comment (ElysiaTanZY) [inline src/main/java/seedu/address/logic/commands/ListEventCommand.java:56]: Delete if not using O:

**Review by ElysiaTanZY — COMMENTED** [review]
Looks good!

File: src/main/java/seedu/address/logic/commands/ListEventCommand.java (hunk 2 above)
Comment (sarahcxh) [reply src/main/java/seedu/address/logic/commands/ListEventCommand.java:56]: Let me try to fix first, if it doesn't work by Friday I'll delete hahaha

**Pull request merged**